README.md

```diff
@@ -30,5 +30,372 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r22.05 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+The current release of the Triton Inference Server is 2.22.0 and
```

Comment:
Remove this paragraph for Readme's in release branches

Reply:
@dzier - I've removed mentioned paragraph